MyProgram.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Threading.Tasks;
+
+namespace MyApp
+{
+    internal class Program
+    {
+        private int counter = 0;
+        private static Actor[] actors =
+        [
+            new Actor { Name = "Alice" },
+            new Actor { Name = "Bob" },
+            new Actor { Name = "Charlie" },
+            new Actor { Name = "Diana" },
+            new Actor { Name = "Ethan" },
+            new Actor { Name = "Fiona" },
+            new Actor { Name = "George" },
+            new Actor { Name = "Hannah" },
+            new Actor { Name = "Ivan" },
+            new Actor { Name = "Julia" }
+        ];
+
+        static async Task Main(string[] args)
+        {
+            if (TryFindActorByName("Julia"))
+            {
+                Console.WriteLine("Julia found");
+            }
+
+            Task.Run(async () =>
```

Comment:
if there's background work going on here, these should be paralleled.

We don't need to wait to fetch Bianca from the server before we fetch Hector from the server.